projects/core/components/scroll-controls/scrollbar.directive.ts

```diff
@@ -45,15 +47,28 @@ export class TuiScrollbarDirective {
         @Inject(WINDOW) private readonly windowRef: Window,
         @Inject(ElementRef) private readonly elementRef: ElementRef<HTMLElement>,
         @Inject(ViewportScroller) private readonly viewportScroller: ViewportScroller,
+        @Inject(TuiScrollbarWrapperDirective)
+        readonly barWrapper: TuiScrollbarWrapperDirective,
     ) {
         const {nativeElement} = this.elementRef;
         const mousedown$ = typedFromEvent(nativeElement, 'mousedown');
         const mousemove$ = typedFromEvent(this.documentRef, 'mousemove');
         const mouseup$ = typedFromEvent(this.documentRef, 'mouseup');
+        const mousedownBarWrapper$ = typedFromEvent(
+            barWrapper.nativeElement,
+            'mousedown',
+        );
 
-        mousedown$
-            .pipe(
+        merge(
+            mousedownBarWrapper$.pipe(
+                preventDefault(),
+                map(event => this.getScrolled(event, 0.5, 0.5)),
+                takeUntil(destroy$),
```

Comment:
Please move `takeUntil` and `tuiZonefree` to the pipe after merge so they are only there once.

Reply:
Ops, my fault. Done it